### PR TITLE
move pep8 changes from PR #69

### PR DESCRIPTION
**Pull request by uralbash**
I deleted pep8 and deform requirements

### CODE REVIEW

File: tests/test_schema.py [hunk 1]
```diff
@@ -949,3 +949,51 @@ def is_equal_schema(self, schema, schema2, schema_path=None):
...
+        self.assertNotIn('foo', schema)
+
+        schema = SQLAlchemySchemaNode(MyClass,
+                                      includes=['id', 'foo', 'job_status'])
+        self.assertEqual(['id', 'job_status'], [x.name for x in schema])
+        self.assertNotIn('foo', schema)
+
+    def test_non_text_includes(self):
+        """
+        TODO: need the doc there
```
Comment (tisdall) [inline tests/test_schema.py]: Either fix this comment or remove it entirely

Comment (uralbash) [inline tests/test_schema.py]: oh I forgot about it

**Comment by uralbash** [comment]
What do you think about this PR, this solves several problems:

* specify the order fields #45 
* add custom fields (it real help me with fk and m2m in new sacrud_deform https://github.com/ITCase/sacrud_deform/blob/master/sacrud_deform/__init__.py)

**Comment by tisdall** [comment]
Okay, it all looks pretty good.  Can you fix the merge conflicts so I can merge it?

**Comment by tisdall** [comment]
Oh..  also add a line in the CHANGES.rst explaining what's changed

**Comment by uralbash** [comment]
I merge it with master and added some lines to CHANGES

**Comment by uralbash** [comment]
Thank you, when you plan to release a new version?

**Comment by tisdall** [comment]
was planning on doing it a few days ago but then some things came up giving me very little time...  I'll see what I can do...

**Comment by uralbash** [comment]
Thank you again

**Pull request merged**